detection_rules/cli_utils.py

```diff
@@ -16,6 +16,7 @@
 import functools
 from . import ecs
 from .attack import matrix, tactics, build_threat_map_entry
+from .config import parse_rules_config
```

Comment:
@eric-forte-elastic I need to pull in changes from https://github.com/elastic/detection-rules/pull/3654 to test import properly.